utilities/zookeeper/R/correctors.county.R

```diff
@@ -80,11 +80,15 @@ zyzzyva_county_corrections <- function(signals_list, ...){
   }
   
   # Multiple Signals
-  if (params$signals_to_correct == c("deaths_incidence_num","confirmed_incidence_num")) {
+  if (all(params$signals_to_correct == c("deaths_incidence_num","confirmed_incidence_num"))|
+      all(params$signals_to_correct == c("confirmed_incidence_num","deaths_incidence_num"))) {
+    
```

Comment:
What if my signals are `deaths`, `cases`, and `hospitalizations` but I only want to correct `deaths` and `cases`? Then no corrections will happen right?